understanding/accessible-authentication-minimum.html

```diff
@@ -125,203 +122,122 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
                </p>
-               <p>While Web sites can use the recognition of objects or of non-text content provided
-                  by the user to meet this Success Criterion, such techniques do not fully support the
-                  cognitive accessibility community and should be avoided if possible. Refer to <a href="accessible-authentication-enhanced">Accessible Authentication (Enhanced)</a> for guidance to be more inclusive and accessible.
+               <p>ウェブサイトは、この達成基準を満たすために利用者が提供するオブジェクト又は非テキストコンテンツの認識を使用できるが、そのような技術は認知アクセシビリティコミュニティを完全にはサポートしておらず、可能な限り避けるべきである。より包括的でアクセシブルにするためのガイダンスについては、<a href="accessible-authentication-enhanced">アクセシブルな認証 (高度)</a> を参照すること。
                </p>
-               <p>This Success Criterion is focused on authentication of existing users. It does <em>not</em> cover creation of a username or initiation of an account.  For many Web sites, establishing
-                  an initial username and credentials may not differ greatly from logging in with that
-                  username. The techniques used to satisfy this criterion (particularly allowing pasting
-                  into inputs and not relying on transcription) can also reduce the cognitive burden
-                  in account creation. However, the focus of the Success Criterion is on reducing the
-                  ongoing need for users to recall previously supplied information each time they log
-                  in or otherwise authenticate to an account.
+               <p>この達成基準は、既存の利用者の認証に焦点を当てたものである。ユーザ名の作成又はアカウントの開始については対象と<em>していない</em>。多くのウェブサイトでは、最初のユーザ名及び資格情報を確立することは、そのユーザ名でログインすることとそれほど変わらない場合がある。この基準を満たすために使用される技術 (特に、入力への貼り付けを許可し、転記に依存しない) によって、アカウント作成時の認知的負担も軽減できる。ただし、この達成基準の焦点は、利用者がログインするたびに、又はその他の方法でアカウントを認証するたびに、以前に提供された情報を思い出す必要性を軽減することにある。
                </p>
                <section id="cog-fun-tests">
                   
-                  <h3>Cognitive Function Tests</h3>
+                  <h3>認知機能テスト</h3>
                   
                   
-                  <p>Remembering a site-specific password is a <a href="#dfn-cognitive-function-test">cognitive function test</a>. Such tests are known to be problematic for many people with cognitive disabilities.
-                     Whether it is remembering random strings of characters, or a pattern gesture to perform
-                     on a touch screen, cognitive function tests will exclude some people. When a cognitive
-                     function test is used, at least one other authentication method must be available
-                     which is not a cognitive function test.
+                  <p>サイト固有のパスワードを記憶することは、<a href="#dfn-cognitive-function-test">認知機能テスト</a>である。そのようなテストは、多くの認知障害のある人々にとって問題となることが知られている。ランダムな文字列を記憶する場合でも、タッチスクリーン上で実行するパターンジェスチャを記憶する場合でも、認知機能テストは一部の人を排除する。認知機能テストを使用する場合、認知機能テストではない他の認証方法が少なくとも一つ使用可能でなければならない。
```

Comment:
最初の文の「a cognitive function test」ですが、その直後に、パスワードを記憶する以外の認知機能テストが挙げられているので、ここでは「認知機能テストの一つ」としてもよいかもしれません。

```suggestion
                  <p>サイト固有のパスワードを記憶することは、<a href="#dfn-cognitive-function-test">認知機能テスト</a>の一つである。そのようなテストは、多くの認知障害のある人々にとって問題となることが知られている。ランダムな文字列を記憶する場合でも、タッチスクリーン上で実行するパターンジェスチャを記憶する場合でも、認知機能テストは一部の人を排除する。認知機能テストを使用する場合、認知機能テストではない他の認証方法が少なくとも一つ使用可能でなければならない。
```